CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Tests now wait for the `getRuntimeContext` request to finish before starting.
+
+### Fixed
+
+- Pickup + Delivery scenario in vtexgame1invoice account.
+- Tests failing due to them trying to type on a disabled element.
```

Comment:
Could you show how to reproduce this bug? Didn't managed to find the tests which had this problem.

Reply:
It was intermittent. Pretty much any first purchase test using credit card could fail due to this bug.